src/nn.h

```diff
@@ -352,6 +364,7 @@ NN_EXPORT int nn_setsockopt (int s, int level, int option, const void *optval,
     size_t optvallen);
 NN_EXPORT int nn_getsockopt (int s, int level, int option, void *optval,
     size_t *optvallen);
+NN_EXPORT int nn_setopt (int option, const void *optval, size_t optvallen);
```

Comment:
Right. I'm going to revert 4331b77 and introduce a configure option to set the fork strategy, but what should I make the default? The current behaviour?